tree/ntuple/v7/src/RPageStorageDaos.cxx

```diff
@@ -40,6 +40,42 @@
 #include <regex>
 
 namespace {
+using AttributeKey_t = ROOT::Experimental::Detail::RDaosContainer::AttributeKey_t;
+using DistributionKey_t = ROOT::Experimental::Detail::RDaosContainer::DistributionKey_t;
+
+/// \brief RNTuple page-DAOS mappings
+enum EDaosMapping { kOClusterDColumn, kOUnique };
```

Comment:
I would probably rename the enumerators in order to clarify what they do (now it seems a bit opaque), i.e. perhaps use `kOidPerCluster` and `kOidPerPage`.